web/docs/formats.md

```diff
@@ -19,8 +19,8 @@ Formats appear as an argument to the [`read`](operators/sources/from.md) and
 [`write`](operators/sinks/to.md) operators:
 
 ```
-read <format> [from <connector>]
-write <format> [to <connector>]
+from <connector> [read <format>]
+to <connector> [write <format>]
```

Comment:
Good catch (the trailing `from/to` doesn't work anymore), but the example does not fit the text above, and the text has wrong links. The example should probably show `from <connector> [read <format>]`, as well as `read <format>` (and the same for `to/write`).